File: android/LiveTales/app/src/main/java/com/zed/livetales/models/book/Page.java
package com.zed.livetales.models.book;

/**
 * Created by jemalpartida on 25/11/2016.
 */

public class Page
{
    public Page(String text, String imagePath, boolean changeImage)
    {
        this.text = text;
        this.imagePath = imagePath;
        this.changeImage = changeImage;
    }

    public String getText()
    {
        return this.text;
    }

    public String getImagePath()
    {
        return this.imagePath;
    }

    public boolean isChangeImage()
    {
        return this.changeImage;
    }


    private String text;
    private String imagePath;
    private boolean changeImage;
}

enum PageType {
    FrontCover,
    Page,
    BackCover
}

File: ios/LiveTales-smartphones/uoat/Page.swift
//
//  Page.swift
//  uoat
//
//  Created by Pyro User on 4/5/16.
//  Copyright © 2016 Zed. All rights reserved.
//

import Foundation

enum PageType
{
    case FrontCover
    case Page
    case BackCover
}

class Page
{
    init( text:String, imagePath:String, changeImage:Bool )
    {
        self.text = text
        self.imagePath = imagePath
        self.changeImage = changeImage
    }
    
    func getText() -> String
    {
        return self.text
    }
    
    func getImagePath() -> String
    {
        return self.imagePath
    }
    
    func isChangeImage() -> Bool
    {
        return self.changeImage
    }

    
    private let text:String
    private let imagePath:String
    private let changeImage:Bool
}
